dd-sdk-android/src/main/kotlin/com/datadog/android/rum/ActivityLifecycleTrackingStrategy.kt

```diff
@@ -0,0 +1,60 @@
+package com.datadog.android.rum
+
+import android.app.Activity
+import android.app.Application
+import android.content.Context
+import android.os.Bundle
+import com.datadog.android.core.internal.utils.devLogger
+
+/**
+ * The ActivityLifecycleTrackingStrategy as an [Application.ActivityLifecycleCallbacks]
+ * based implementation of the [TrackingStrategy].
```

Comment:
`based` → `base`

Reply:
Actually my intention was to say "based" meaning that this strategy is based on that  implementation.